package.json

```diff
@@ -5,6 +5,9 @@
     "author": "Fredrik Jonsson",
     "license": "GPL-2.0",
     "babel": {
+        "exclude": [
+            "**/*.min.js"
+        ],
```

Comment:
Required for nprogress.js to work.

Reply:
And not really any point in processing minified libs.

Reply:
Right!